packages/varlet-ui/src/date-picker/date-picker.less

```diff
@@ -48,7 +49,7 @@
   font-size: var(--date-picker-font-size);
   position: relative;
   min-width: var(--date-picker-min-width);
-  height: var(--date-picker-height);
+  // height: var(--date-picker-height);
```

Comment:
There will be an inappropriate height.

![image](https://github.com/varletjs/varlet/assets/34639100/b1a7b195-a0e4-4015-bbef-e29072af5080)

Reply:
If using a fixed height may not be conducive to the expansion of custom action panels, then I'll try fixing the body height of the date-picker

Reply:
Any way can be, but the height can't change, you can see [this](https://play.vuetifyjs.com/#eNptj0EOwiAQRa9Cxq2liXajwSaew7pAOibYOhCgNcZ4dym1SRtdAX8eH97pBd6p/Ggt7zuEPYiAd9vKgGVFjIk+U4aC1IQuBSly5sFunQ/6+jxU4K1UmElnOqor+FKJq2NNZrVq0DFssZdBG4o3NsWMi+T0JFtJNSB+NkxFl0ClkqSwFfl4+gOY5nco8oXOGC0+NlnFNGqN0nE/t16UwHsNW77lOxjWghdw/gD2tGMR)

https://github.com/varletjs/varlet/assets/34639100/2285131d-9b66-4930-958b-a246444f9a31

Reply:
I get it